core/conn/spj_init/README.md

```diff
@@ -0,0 +1,24 @@
+##How to build and initialize default SPJs
+###Build
+```  
+cd $MY_SQROOT/../conn/spj_init    
+make clean && make all  
+```
+###Initialization
+```  
+cds  
+./init_spj.sql  
+```  
+###SPJ Example
+```
+DEFAULT_SPJ.HELP(INOUT COMMANDNAME VARCHAR)  
+E.g.   
+trafci>set param ?p1 help;  
+trafci>call DEFAULT_SPJ.HELP(?p1)  
+PUT - Upload a JAR. SHOWDDL PRODURE DEFAULT_SPJ.PUT for more info.  
+LS - List JARs. SHOWDDL PRODURE DEFAULT_SPJ.LS for more info.  
+LSALL - List all JARs. SHOWDDL PRODURE DEFAULT_SPJ.LSALL for more info.  
+RM - Remove a JAR. SHOWDDL PRODURE DEFAULT_SPJ.RM for more info.  
+RMREX - Remove JARs by a perticular pattern. SHOWDDL PRODURE DEFAULT_SPJ.RMREX for more info.  
+GETFILE - Upload a JAR. SHOWDDL PRODURE DEFAULT_SPJ.GETFILE for more info.  
+```
```

Comment:
Do you need to run as the Trafodion ID to enable this feature?  If someone like Webroot wants to initialize this, how would this be done?  I assume someone would need to run the init_sqj script.  Will this be automated somehow?  Can someone other than the Trafodion ID perform this operation if they have the correct database privileges?